src/lando/main/scm/git.py

```diff
@@ -210,6 +210,14 @@ def apply_patch(
 
                     raise exc
 
+    def apply_patch_bytes(self, patch_bytes: bytes):
+        """Apply the given `git format-patch` to the repo directly."""
+        with tempfile.NamedTemporaryFile(mode="wb", suffix=".patch") as tmp_file:
+            tmp_file.write(patch_bytes)
+            tmp_file.flush()
+
+            self._git_run("am", tmp_file.name, cwd=self.path)
```

Comment:
As you suggested, adding `--keep-cr` is needed.

```suggestion
            self._git_run("am", "--keep-cr", tmp_file.name, cwd=self.path)
```